romanesco/format/__init__.py

```diff
@@ -86,19 +86,9 @@ def converter_path(source, target):
     :returns: An ordered list of the analyses that need to be run to convert from
     ``source`` to ``target``.
     """
-    def get_edge_analysis(source, target):
-        """Return the analysis associated with an edge in the conversion graph,
-        given the source Validator and the target Validator.
-        """
-        for (u, v, data) in conv_graph.edges([source], data=True):
-            if v == target:
-                return data
-
-        return {}
-
     # These are to ensure an exception gets thrown if source/target don't exist
-    get_validator(source)
-    get_validator(target)
+    get_validator_analysis(source)
+    get_validator_analysis(target)
```

Comment:
maybe consider wrapping these in a try/catch  and throwing a NetworkXNoPath exception instead of the generic Exception()  thrown by get_validator_analysis ?